### PR TITLE
Documentation: Add `adm:pass` to replication endpoint URL

### PR DESCRIPTION
**Pull request by jiahuili430**
## Overview

When using replication, we need to specify a username and password (`adm:pass` / `admin:password`) for the replication endpoint URL, otherwise a 401 unauthorized error will be thrown.

## Related Issues or Pull Requests

Fix https://github.com/apache/couchdb/issues/4127

## Checklist

- [ ] Code is written and works correctly
- [ ] Changes are covered by tests
- [ ] Any new configurable parameters are documented in `rel/overlay/etc/default.ini`
- [x] Documentation changes were made in the `src/docs` folder
- [ ] Documentation changes were backported (separated PR) to affected branches

### CODE REVIEW

**Comment by big-r81** [comment]
Hi @jiahuili430,

thanks for taking a look and fixing this. Do you only concentrate to fix the `/_replicate` endpoint, or do you search for other occurences of this behavior in the docs as Nick suggested?

**Comment by jiahuili430** [comment]
I only checked the `1.7.4 Replication` and `2. Replication` sections.

File: src/docs/src/replication/replicator.rst [hunk 1]
```diff
@@ -252,16 +252,16 @@ normal.
...
 the following order:
 
 .. code-block:: javascript
 
     {
         "_id": "my_rep",
-        "source": "http://myserver.com/foo",
-        "target":  "http://user:pass@localhost:5984/bar",
-        "create_target":  true,
+        "source": "http://adm:pass@myserver.com/foo",
```
Comment (nickva) [inline src/docs/src/replication/replicator.rst:262]: In a few cases, where we have a remote endpoint like "myserver.com" we may want to use a different password user combination to illustrate that those endpoint may have a completely different set of credentials. So in this case we could have source have a `user:password` for instance

Comment (big-r81) [reply src/docs/src/replication/replicator.rst:262]: Yeah, that were my thoughts too, but great work @jiahuili430 so far, it's almost ready already...

Comment (jiahuili430) [reply src/docs/src/replication/replicator.rst]: Changed, thanks for the review!

**Review by big-r81 — APPROVED** [review]
+1

Great work @jiahuili430 and thank you 🙏 for your contribution!

**Pull request merged**